src/JustEat.StatsD/TagsStyle.cs

```diff
@@ -0,0 +1,35 @@
+namespace JustEat.StatsD
+{
+    /// <summary>
+    /// An enumeration defining the tagging formats supported.
+    /// <para />
+    /// Disabled is the default.
+    /// </summary>
+    public enum TagsStyle
```

Comment:
As noted earlier, let's remove the need for this enum so it's extensible without needing a whole new release for a new provider.

Reply:
Done.